backend/src/main/java/ubb/courses/backend/services/course/CourseService.java

```diff
@@ -27,4 +26,7 @@ public CourseService(CourseRepository courseRepository) {
     public Course getCourseById(Integer id) {
         return this.courseRepository.findById(id).orElseThrow(() -> new CourseException("Course not found!"));
     }
+
+    @Override
+    public void addCourse(Course c){this.courseRepository.save(c);}
```

Comment:
Can you make this transactional ? We don't want corrupted data inside the database

Reply:
and name the variable c as course